docs/concepts/capabilities.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Capabilities'
+icon: 'ti-anchor' # themify icon pack : https://themify.me/themify-icons
+description: 'Application declaration, deployment, and management'
+sidebar_position: 1
+type: 'docs'
+---
+
+## Overview
+
+A ***capability** is a pluggable piece of functionality for an application. More specifically, capabilities are common, abstracted functionalities&mdash;usually the sort of utilities that aren't core business logic for a given application, such as...
+
+* HTTP Client & Server
+* Logging
+* Key-Value Storage
+* Messaging
+* Blob Storage
+* Secrets
+
+You can find a complete list of first-party capabilities in the [Capability Catalog](/docs/capabilities/index.mdx).
+
+## How capabilities work
+
+Capabilities are enacted by an **interface** and one or more swappable **capability providers**. A provider delivers a specific implementation of a given capability by communicating with other entities in the common tongue of a standard API. 
+
+For example...
+
+* The [Redis key-value storage provider](https://github.com/wasmCloud/wasmCloud/pkgs/container/keyvalue-redis) delivers the key-value capability to application components over the standard `wasi:keyvalue` API. 
+* If the development team chooses to use Vault for key-value storage instead, they can swap out the Redis provider for the [Vault provider](https://github.com/wasmCloud/wasmCloud/pkgs/container/keyvalue-vault). 
+* The Vault capability provider will deliver a different implementation of the same capability *over the same API*.  
+
+## Creating your own capabilities
+
+wasmCloud supports custom capabilities&mdash;use our provider SDKs for [Rust](https://crates.io/crates/wasmcloud-provider-sdk) and [Go](https://github.com/wasmCloud/provider-sdk-go) to create your own providers, and write your own interfaces in the [WebAssembly Interface Type (WIT)](/docs/concepts/interfaces) interface description language. For more information, see...
```

Comment:
```suggestion
wasmCloud supports custom capabilities&mdash;use our provider SDKs for [Rust](https://crates.io/crates/wasmcloud-provider-sdk) and [Go](https://github.com/wasmCloud/provider-sdk-go) to create your own providers, and write your own interfaces in the [WebAssembly Interface Type (WIT)](/docs/concepts/interfaces) interface description language. 

For more information, see...
```

Also this might be a good place to link to the provider examples:

https://github.com/wasmCloud/wasmCloud/tree/main/examples/golang/providers/custom-template
https://github.com/wasmCloud/wasmCloud/tree/main/examples/rust/providers/custom-template

 (or even suggest them as ways to start w/ `wash new`?